gopherage/junit/calculation/coverage.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package calculation
+
+import (
+	"fmt"
+)
+
+// coverage stores test coverage summary data for one file
+type coverage struct {
+	Name          string
+	nCoveredStmts int
+	nAllStmts     int
+}
+
+// Percentage returns the percentage of statements covered
+func (c *coverage) Percentage() string {
+	ratio, err := c.ratio()
+	if err == nil {
+		return fmt.Sprintf("%.1f", ratio*100)
+	}
+
+	return ""
+}
+
+func (c *coverage) ratio() (float32, error) {
```

Comment:
Would it make sense for this to consider a zero-line file to be 100% covered instead of making every caller of this handle an error case?